src/types/general.ts

```diff
@@ -95,3 +95,12 @@ export type OrderLogProps = {
   orderDiscounts: OrderDiscountProps[]
   orderExecutors: string[]
 }
+
+export type PermissionGroupProps = {
+  id: string
+  name: string
+  permissionGroupPermission?: {
+    id: string
+    permissionId: string
+  }[]
```

Comment:
in our situation, it could be
```ts
type PermissionGroupProps = {
  id: string
  name: string
  permissionIds: string[]
}
```

Reply:
type Member = {
    id: string
    name: string
    gender: "male" | "female"
}

type ProgramProps
type ProjectProps
type ActivityProps

type MemberProps = Pick<Member, "name" | "id"> & {
    isMale: boolean
}

const MemberInfo<MemberProps> = () => {
}